### PR TITLE
[PB-9370] Added sample sageUK/Bellevie config. Parameterized scope for Docker image.

### PR DESCRIPTION
**Pull request by egorvel**
* Added ep.config-bellevie.json for easy configuration on training VM.
* Added a docker argument to allow the scope `vestri` to be overridden.
* Duplicated the localization files from CA to GB.

### CODE REVIEW

**Comment by nelsonq** [comment]
```  
  {
      "value": "EUR",
      "name": "Euro"
    }
```
looks like missing in the `src/ep.config-bellevie.json` file

**Comment by egorvel** [comment]
> ```
>   {
>       "value": "EUR",
>       "name": "Euro"
>     }
> ```
> 
> looks like missing in the `src/ep.config-bellevie.json` file

@nelsonq It is by design, CAD/ EUR should not be available.

File: src/ep.config-bellevie.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
+{
+  "cortexApi": {
+    "path": "/cortex",
+    "scope": "vestri",
```
Comment (ivanjensen-ep) [inline src/ep.config-bellevie.json:4]: change this to `sageUK`

File: src/ep.config-bellevie.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    "pathForProxy": "http://localhost:9080",
+    "reqTimeout": "30000"
+  },
+  "supportedLocales": [
+    {
+      "value": "en-GB",
+      "name": "English UK"
+    },
+    {
+      "value": "fr-FR",
```
Comment (ivanjensen-ep) [inline src/ep.config-bellevie.json:14]: Remove french

**Review by ivanjensen-ep — APPROVED** [review]

**Comment by nelsonq** [comment]
LTGM

**Pull request merged**